### PR TITLE
Fix Polish plurals

### PR DESCRIPTION
**Pull request by soshial**

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -22241,7 +22241,9 @@
     mr = %d वस्तू
     nb = %d sted
     nl = %d plaats
-    pl = %d miejsce
+    pl:one = %d miejsce
```
Comment (biodranik) [inline data/strings/strings.txt:22244]: This word is used for tracks and bookmarks, so miejsce is not a correct translation. Also see https://github.com/organicmaps/organicmaps/pull/8245

File: data/strings/strings.txt [hunk 2]
```diff
@@ -22307,7 +22309,9 @@
     mr:other = %d ठिकाणे
     nb = %d steder
     nl = %d plaatsen
-    pl = %d miejsc
+    pl:one = %d miejsce
+    pl:few = %d miejsca
+    pl:many = %d miejsc
```
Comment (biodranik) [inline data/strings/strings.txt:22314]:
```suggestion
    pl:few = %d miejsca
    pl:many = %d miejsc
    pl:one = %d miejsce
```

File: data/strings/strings.txt [hunk 3]
```diff
@@ -22369,7 +22373,9 @@
     mr = %d ट्रॅक
     nb = %d stier
     nl = %d tracks
-    pl = %d tras
+    pl:one = %d trasa
+    pl:few = %d trasy
+    pl:many = %d tras
```
Comment (biodranik) [inline data/strings/strings.txt:22378]:
```suggestion
    pl:few = %d trasy
    pl:many = %d tras
    pl:one = %d trasa
```

Our strings validator requires strings to be sorted alphabetically.

**Review by biodranik — COMMENTED** [review]
Thanks! Left a few comments.

File: data/strings/strings.txt (hunk 1 above)
Comment (soshial) [reply data/strings/strings.txt:22244]: Should I wait till #8245 will be merged? Will there be conflicts?

Comment (biodranik) [reply data/strings/strings.txt:22244]: Don't wait, conflicts can be resolved.

Comment (soshial) [reply data/strings/strings.txt:22244]: Changed to objects

File: data/strings/strings.txt (hunk 2 above)
Comment (soshial) [reply data/strings/strings.txt]: Changed order as you showed

File: data/strings/strings.txt (hunk 3 above)
Comment (soshial) [reply data/strings/strings.txt]: Changed order as you showed

**Review by biodranik — APPROVED** [review]
Dziękuję!

**Pull request merged**